docs/cluster-configuration.md

```diff
@@ -249,6 +249,14 @@ See [below](#configure-application-aware-quota-aaq) for Application Aware Quota
 
 **Default**: `false`
 
+### primaryUserDefinedNetworkBinding Feature Gate
+Set the `primaryUserDefinedNetworkBinding` feature gate to deploy the needed configurations for kubevirt users, so they 
+can bind their VM to a user-defined network (UDN) on the VM's primary interface.
+
```

Comment:
do we have some kind of documentation we can add as a reference? I'm afraid this description is a bit crypt.

Reply:
@maiqueb do we?

Reply:
what documentation are you hoping for @nunnatsa ? 

I'm afraid all we have is the [openshift enhancement document](https://github.com/openshift/enhancements/pull/1623).

Reply:
maybe worth to add the env var that can be set in order to determine non default value
(this what we are doing for example on CNAO readme)
together with the current info, it can give a simple user guide (please consider adding few words that will make it user guide in case it miss)

Reply:
> maybe worth to add the env var that can be set in order to determine non default value (this what we are doing for example on CNAO readme) together with the current info, it can give a simple user guide (please consider adding few words that will make it user guide in case it miss)

Not sure what you mean here. what env var? This FG is set by HCO CR, and is set to `false` by default.. In order to activate it, no env var is required - only changing the FG value to `true`. 

> also good to have please on PR desc the important details, such as the feature gate name and so on

DONE

Reply:
> I'm afraid all we have is the [openshift enhancement document](https://github.com/openshift/enhancements/pull/1623).

@nunnatsa will that be sufficient?

Reply:
> Not sure what you mean here. what env var? This FG is set by HCO CR, and is set to `false` by default.. In order to activate it, no env var is required - only changing the FG value to `true`.
> 

I meant the one that allows to override the image, but no biggie

Reply:
ah.. that's not something a user should be exposed to imo.